pubsub/gcloud/aio/pubsub/subscriber_client.py

```diff
@@ -15,12 +20,30 @@ def __init__(self, **kwargs) -> None:
     from google.api_core import exceptions
     from google.cloud import pubsub
     from google.cloud.pubsub_v1.subscriber.message import Message
-    from google.cloud.pubsub_v1.subscriber.scheduler import Scheduler
-    from google.cloud.pubsub_v1.types import FlowControl
 
+    from .subscriber_message import SubscriberMessage
     from .utils import convert_google_future_to_concurrent_future
 
 
+    class FlowControl():
+        def __init__(self,
+                     max_bytes: int = 100 * 1024 * 1024,
+                     max_messages: int = 1000,
+                     max_lease_duration: int = 1 * 60 * 60,
+                     max_duration_per_lease_extension: int = 0) -> None:
+            self.max_bytes = max_bytes
+            self.max_messages = max_messages
+            self.max_lease_duration = max_lease_duration
+            self.max_duration_per_lease_extension = (
+                max_duration_per_lease_extension)
```

Comment:
Could you add a bit of documentation to describe each of the fields briefly?

Reply:
Agreed -- might be nice to link to the Google code for this class, at least, since IIRC they have some light docs on the meaning of each of these fields inline.

Reply:
Okay -- so -- I think _not wrapping_ Google's class is more problematic than just wrapping it -- so my solution here is twofold: (1) wrap the class with a local passthru; (2) link to the docs for the valid fields; (3) as a convenience, list the valid fields and default values;